include/zephyr/bluetooth/audio/has.h

```diff
@@ -67,6 +67,26 @@ enum bt_has_capabilities {
 	BT_HAS_PRESET_SUPPORT = BIT(0),
 };
 
+/* Preset event types */
+enum bt_has_preset_event_type {
+	BT_HAS_PRESET_DELETE,
+	BT_HAS_PRESET_AVAILABLE,
+	BT_HAS_PRESET_UNAVAILABLE,
+	BT_HAS_PRESET_RECORD_UPDATE,
+	BT_HAS_PRESET_RECORD_READ_RSP,
```

Comment:
Should these be past tense? 

e.g. if the type is `BT_HAS_PRESET_DELETE`, doesn't that imply that the preset was _delete*d*_?

Reply:
Could be